language-support/hs/bindings/src/DA/Ledger/Types.hs

```diff
@@ -103,6 +103,11 @@ data Command
       }
     deriving (Eq,Ord,Show)
 
+data Deduplication
```

Comment:
Can we go with `DeduplicationPeriod`?

Reply:
Good point, should've cleaned and used the consistent naming everywhere